api/pkg/transformer/feast/transformer_test.go

```diff
@@ -475,3 +476,510 @@ func TestTransformer_Transform(t *testing.T) {
 		})
 	}
 }
+
+func Test_buildEntitiesRequest(t *testing.T) {
```

Comment:
Can you add test that involves 3 entities? Want to see how it generalize

Reply:
Already added: https://github.com/gojek/merlin/pull/73/files/232cd0692aa1f1d2902c67757bae766b8681620c#diff-a6950993560c99c56bcc7cc2aaa7e80747301e01c16056044ee0703151f3022dR716-R914

I also add test for 4 entities: https://github.com/gojek/merlin/pull/73/files/232cd0692aa1f1d2902c67757bae766b8681620c#diff-a6950993560c99c56bcc7cc2aaa7e80747301e01c16056044ee0703151f3022dR916